Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -425,8 +426,12 @@ open class ChartDataSet: ChartBaseDataSet
     // TODO: This should return the removed entry to follow Swift convention.
     open override func removeFirst() -> Bool
     {
-        let entry: ChartDataEntry? = values.isEmpty ? nil : values.removeFirst()
-        return entry != nil
+        guard !values.isEmpty else { return false }
+
+        values.removeFirst()
+        calcMinMax()
```

Comment:
You find this bug for not adding  calcMinMax here? Do we have an issue to track or someone reported